src/model/tool.rs

```diff
@@ -157,6 +167,50 @@ mod tests {
         );
     }
 
+    #[test]
+    fn multiple_asset_found() {
+        let asset_name = "asset";
+
+        let tool_info = ToolInfo {
+            owner: "owner".to_string(),
+            repo: "repo".to_string(),
+            exe_name: "exe".to_string(),
+            tag: ToolInfoTag::Latest,
+            asset_name: AssetName {
+                linux: Some(asset_name.to_string()),
+                macos: Some(asset_name.to_string()),
+                windows: Some(asset_name.to_string()),
+            },
+        };
+
+        let assets = vec![
+            Asset {
+                id: 1,
+                name: asset_name.to_string(),
```

Comment:
Can we change asset names in this test to have unique test names in each element?
Just to test that the function actually returns different assets from every item and not just replicates the same asset name several times. An easy way to increase test coverage without adding new tests, just changing existing ones 🙂 
```suggestion
                name: "asset_1".to_string(),
```